CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * [CHANGE] Blocks uploaded by ingester no longer contain `__org_id__` label. Compactor now ignores this label and will compact blocks with and without this label together. `mimirconvert` tool will remove the label from blocks as "unknown" label. #1972
 * [CHANGE] Querier: deprecated `-querier.shuffle-sharding-ingesters-lookback-period`, instead adding `-querier.shuffle-sharding-ingesters-enabled` to enable or disable shuffle sharding on the read path. The value of `-querier.query-ingesters-within` is now used internally for shuffle sharding lookback. #2110
 * [CHANGE] Memberlist: `-memberlist.abort-if-join-fails` now defaults to false. Previously it defaulted to true. #2168
+* [FEATURE] Ingester: support for out-of-order samples ingestion. (Feature description to be completed) (PR TBD).
```

Comment:
We will add this entry with the PR that brings in this support. We can skip it here.

Reply:
https://github.com/grafana/mimir/pull/2187

Reply:
https://github.com/grafana/mimir/pull/2183/commits/4188e729e4c83add4ae845294c52fdcad93950f9